src/mcp/client.js

```diff
@@ -0,0 +1,46 @@
+// src/mcp/client.js
+import 'dotenv/config';
+import fetch from 'node-fetch';
+
+let requestId = 1;
+
+export async function callMCP(method, extraHeaders = {}, params = {}) {
+  if (!method) throw new Error('method is required');
+
+  const bodyObj = {
+    jsonrpc: '2.0',
+    id: requestId++,
+    method,
+    params,
+  };
+
+  const headers = {
+    'Content-Type': 'application/json',
+    'x-api-key': process.env.MCP_API_KEY,
```

Comment:
I'd update the readme to call out those env variables and how to get them as well